pom.xml

```diff
@@ -122,6 +122,7 @@
         <influxdb.version>${influx-java-driver-version}</influxdb.version>
         <jackson.version>2.16.1</jackson.version><!-- @sync io.quarkus:quarkus-bom:${quarkus.version} dep:com.fasterxml.jackson.core:jackson-core -->
         <jakarta.jms-api.version>${jakarta-jms-api-version}</jakarta.jms-api.version>
+        <jasypt.version>1.9.3</jasypt.version><!-- Remove once we are on Quarkus CXF 3.8.4 where jasypt is managed -->
```

Comment:
Might be better to stay aligned with Camel, even if it's just a temporary thing:

```suggestion
        <jasypt.version>${jasypt-version}</jasypt.version><!-- Remove once we are on Quarkus CXF 3.8.4 where jasypt is managed -->
```